packages/cli/src/commands/lockedgold/register.ts

```diff
@@ -1,11 +1,10 @@
-import { BondedDeposits } from '@celo/walletkit'
-
+import { LockedGold } from '@celo/walletkit'
 import { BaseCommand } from '../../base'
 import { displaySendTx } from '../../utils/cli'
 import { Flags } from '../../utils/command'
 
 export default class Register extends BaseCommand {
-  static description = 'Register an account for bonded deposit eligibility'
+  static description = 'Register an account for Locked Gold'
```

Comment:
Wash changing the wording a conscious choice?

Reply:
yes